docs/pages/admin-guides/access-controls/guides/hardware-key-support.mdx

```diff
@@ -257,7 +257,7 @@ Make sure that the touch and PIN policy satisfy the hardware key requirement for
 
 ### `ERROR: private key policy not met`
 
-This error is returned by the Auth and Proxy services if a user does not meet the required private key policy.
+This error is returned by the Auth Service and Proxy Service services if a user does not meet the required private key policy.
```

Comment:
Another issue with "service services" here, I think it'd be a good idea to go through the whole branch and see if there are other places which do this as there's 263 files changed in this PR.

Reply:
Fixed - previous entries:

```
/Users/mmcallister/Teleport/teleport/docs [0824_prose_audit]
% grep -inr 'service service' */| wc -l
      10
```

Corrected them manually.

```
% grep -inr 'service services' */|wc -l
       0
```